### DISCUSSION TITLE
Tidak bisa knit keseluruhan. Namun, run code masing-masing bisa

### DISCUSSION THREAD

**Discussion by AngelMartha** [teamalgoritma/community#736] [Q&A] [answered]
![Screenshot (1253)](https://github.com/teamalgoritma/community/assets/127745812/618a3923-71e8-47fe-ab63-a34d62ad5f17)

**Comment by divakartika** [comment, marked as answer]
Dear @AngelMartha ,

Error message menunjukkan bahwa pada line 293 tidak ada object `test`. Apakah pada chunk-chunk sebelumnya sudah dibuat object `test`? Jika sudah terbuat bisa dilampirkan kembali screenshot bagian pembentukan object `test`.

**Comment by kevbow** [comment]
Halo @AngelMartha, untuk menjaga agar forum diskusi tetap berjalan kondusif, kami izin melakukan lock/close diskusi ini. Semoga dapat dipahami.
